Packs/CortexXDR/ReleaseNotes/4_10_29.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Fixed an issue in the **fetch-incidents** and the **xdr-get-incident-extra-data** command where the `Endpoint.Hostname` value returned to the context was invalid.
```

Comment:
I would give some more information, like : before we returned "endpoint:id", now w return them seperate.

Reply:
i have updated the release-notes to be a bit different (also from your suggestion), let me know what you think